playbook.yaml

```
@@ -0,0 +1,92 @@
+- name: Install ToDo App on new web servers
+  hosts: webservers
+  remote_user: ec2-user
+  vars_prompt:
+    - name: TRELLO_API_KEY
+      prompt: What is your Trello API key?
+      private: false
+
+    - name: TRELLO_API_TOKEN
+      prompt: What is your Trello API token?
+      private: false
+
+    - name: TRELLO_BOARD_ID
+      prompt: What is your Trello Board ID?
+      private: false
+
+  tasks:
+    - name: Install git
+      yum:
+        name: git
+        state: present
+      become: yes
+
+    - name: Install python
+      yum:
+        name: python3
+        state: present
+      become: yes
+
+    - name: Install node
+      yum:
+        name: nodejs
+        state: present
+      become: yes
```

Minor but worth noting that we can combine these if we want to:
```suggestion
    - name: Install git
      yum:
        name: 
          - git
          - python3
          - nodejs
        state: present
      become: yes
```